pkgs/development/python-modules/python-sat/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildPythonPackage, fetchPypi, lib, six, pypblib, pytest }:
+
+buildPythonPackage rec {
+	pname = "python-sat";
+  version = "0.1.6.dev6";
+
+  src = fetchPypi {
+		inherit pname version;
+    sha256 = "7v8IP0JK3Xjmf4uuY/Z25aRINU15A2H0hNGFI8E5Or0=";
```

Comment:
There's still a preference to having base32 hashes within nixpkgs to ease the upgrade process for users with holder versions of the nix cli.

You can use `nix-hash --to-base32 <hash>` to convert the hash into the related base32 hash.

For more details refer to: https://github.com/NixOS/nixpkgs/pull/89308#issuecomment-636927054 #89275